packages/compass-connection-import-export/src/components/export-modal.tsx

```diff
@@ -33,14 +30,12 @@ export function ExportConnectionsModal({
   favoriteConnections,
```

Comment:
If we're starting to use hooks here proper, then we probably can use connection repository to get this info from service directly too instead of passing it here?